Vice.js

```diff
@@ -0,0 +1,438 @@
+{
+	"translatorID": "131310dc-854c-4629-acad-521319ab9f19",
+	"label": "Vice",
+	"creator": "czar",
+	"target": "^https?://((www|broadly|creators|i-d|amuse-i-d|impact|motherboard|munchies|news|noisey|sports|thump|tonic|waypoint)\\.)?vice\\.com",
```

Comment:
I'd be OK with the simpler `^https://(.+?\\.)vice\\.com` unless you need to specifically exclude certain subdomains